README.md

```diff
@@ -35,92 +39,49 @@ Yellow  GND 5  ---       +--+
                +---------+
 ```
 
-### Software Installation
+## Software Installation
 
-Requires having first installed ESPHome. See Getting Started at <https://esphome.io/index.html> for ESPHome installation instructions.
+Setup can be done from the command line using the esphome docker image, or via a (mostly) graphical interface by using an esphome dashboard installation; see Getting Started at <https://esphome.io/index.html> for full ESPHome installation and usage instructions. Initial installation must be done over USB. After this, updates and configuration changes can be made entirely over WiFi.
 
-The below econet based devices are supported.  Each has its own `yaml` configuration file in this repo that defines the applicable Home Assistant Sensors and Controls.  These are:
+Installation is a two step process:
 
-#### Tankless Water Heater
+1. Customize the basic configuration yaml for your device and environment
+2. Deploy to the device via command line or ESPHome Dashbaord
 
-- `econet_tankless_water_heater.yaml`
+### Step 1: Customizing the esphome-econet yaml for Your Device
 
-#### Heat Pump Water Heater
+Before installing, you will need to create a customized yaml for your device/environment. Start with either the [example-esp32.yaml](https://github.com/stockmopar/esphome-econet/blob/main/example-esp32.yaml) or [example-esp8266.yaml](https://github.com/stockmopar/esphome-econet/blob/main/example-esp8266.yaml) files as a template, depending on your chosen hardware.
 
-- `econet_heatpump_water_heater.yaml`
-
-#### HVAC
+At a minimum, you will need to either [put in your own WiFi network details](https://esphome.io/components/wifi) or provide a `secrets.yaml` file with the `wifi_ssid` and `wifi_password` fields configured. An example `secret.yaml` would look like:
+      ```yaml
```

Comment:
Can you make sure this renders correctly?
GitHub preview shows it as a single line https://github.com/stockmopar/esphome-econet/blob/54ff06667170875f0a90986b0bab50855e7250d3/README.md

Reply:
Need to add a newline. Fixed.